kippo/accounts/admin.py

```diff
@@ -162,15 +163,24 @@ def get_github_organizations(self, obj):
 class PersonalHolidayAdmin(AllowIsStaffAdminMixin, admin.ModelAdmin):
     list_display = ("user", "is_half", "day", "duration")
 
+    def get_form(self, request, obj=None, **kwargs):
+        form = super().get_form(request, obj, **kwargs)
+        if not request.user.is_superuser:
+            form.base_fields["user"].widget = forms.HiddenInput()
+            form.base_fields["user"].initial = request.user
+        return form
+
     def save_model(self, request, obj, form, change):
-        if getattr(obj, "pk", None) is None:
+        if not request.user.is_superuser:
             obj.user = request.user
         obj.save()
 
     def get_queryset(self, request):
```

Comment:
Make a testcase for this too.

Confirm: 

- case where user is superuser
- case where is NOT superuser and NO orginizationmembership
- case where is NOT superuser and HAS organizationalmembership